src/pres/loader/Filters.js

```diff
@@ -22,10 +22,12 @@ export default class User extends React.Component {
 
     constructor(props) {
         super(props)
+
+        let isAdvancedFiltersOpen = new URLSearchParams(window.location.search).get("advanced") === 'true'
```

Comment:
You could argue (I don't have a strong opinion on this, and this approach was simplest -- flagging it here just in case) that it is unintuitive that `advanced=1` is treated as false.